manifests/authentication/certificate.pp

```diff
@@ -96,6 +96,16 @@
         require => Package[$lightblue::eap::package_name],
     }
 
+    # create ldap-healthcheck-config.json file required to initialize RolesProvider
+    file { "${directory}/ldap-healthcheck-config.json":
```

Comment:
Defined a variable $module_directory = '/usr/share/jbossas/modules/com/redhat/lightblue/main' to avoid dependency on other modules. It is similarly defined in module.pp